test/functional/terminal/job_spec.lua

```diff
@@ -31,9 +30,6 @@ describe('job handler', function()
         elseif 'stderr' ==# a:event
           put =join(a:data,"\n")
         elseif 'exit' ==# a:event
-          " Length of received data.
-          call setline('.', 'bytes: '.len(join(g:job_stdout[0:-2],"\n")))
```

Comment:
> Because the job handlers here are blindly joining a:data without regards to whether a:data continues a previous partial result.

Then the help should be clarified.  I've read ZyX's response there and the help a few times now and I can't fully understand ZyX's description nor see anything in the help that agrees with what I think he's trying to state.